### DISCUSSION TITLE
Error when installing cask biglybt

### DISCUSSION THREAD

**Discussion by sukucorp** [Homebrew/discussions#2874] [Casks]
I am getting the following error when installing cask biglybt.

Running `brew update --preinstall`...
==> Cask::Installer#install
==> Printing caveats
==> Caveats
biglybt requires Java 8+. You can install the latest version with:
  brew install --cask temurin

==> Cask::Installer#fetch
==> Downloading https://github.com/BiglySoftware/BiglyBT/releases/download/v2.9.0.0/GitHub_BiglyBT_Installer.dmg
/opt/pkgmgr/homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/3.3.12-23-g05b5a2c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 11.6.2\)\ curl/7.64.1 --header Accept-Language:\ en --retry 3 --location --silent --head --request GET https://github.com/BiglySoftware/BiglyBT/releases/download/v2.9.0.0/GitHub_BiglyBT_Installer.dmg
Already downloaded: /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
==> Checking quarantine support
/usr/bin/xattr -h
/usr/bin/swift /opt/pkgmgr/homebrew/Library/Homebrew/cask/utils/quarantine.swift
==> Quarantine is available.
==> Verifying Gatekeeper status of /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
/usr/bin/xattr -p com.apple.quarantine /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
==> /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg is quarantined
==> Verifying checksum for cask 'biglybt'
hdiutil imageinfo -format /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
==> Installing Cask biglybt
==> Cask::Installer#stage
==> Extracting primary container
==> Using container class UnpackStrategy::Dmg for /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
hdiutil attach -plist -nobrowse -readonly -mountrandom /private/tmp/d20220127-4682-jsn84q /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
find . -print0
mkbom -s -i /private/tmp/20220127-4682-1g4t78p.list -- /private/tmp/20220127-4682-jaxxta.bom
ditto --bom /private/tmp/20220127-4682-jaxxta.bom -- /private/tmp/d20220127-4682-jsn84q/dmg.f6yy2K /private/tmp/d20220127-4682-cu50ed
diskutil info -plist /private/tmp/d20220127-4682-jsn84q/dmg.f6yy2K
diskutil eject /private/tmp/d20220127-4682-jsn84q/dmg.f6yy2K
cp -pR /private/tmp/d20220127-4682-cu50ed/LICENSE.txt /opt/pkgmgr/homebrew/Caskroom/biglybt/2.9.0.0/LICENSE.txt
cp -pR /private/tmp/d20220127-4682-cu50ed/BiglyBT\ Installer.app/. /opt/pkgmgr/homebrew/Caskroom/biglybt/2.9.0.0/BiglyBT\ Installer.app
chmod -Rf +w /private/tmp/d20220127-4682-cu50ed
==> Verifying Gatekeeper status of /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
/usr/bin/xattr -p com.apple.quarantine /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
==> /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg is quarantined
==> Propagating quarantine from /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg to /opt/pkgmgr/homebrew/Caskroom/biglybt/2.9.0.0
/usr/bin/xattr -p com.apple.quarantine /Users/admin/Library/Caches/Homebrew/downloads/aad265abfc5118129c2fa1de4bbf05e9fd4cbeb10429849b85fa7b0f74e13d1b--GitHub_BiglyBT_Installer.dmg
/usr/bin/xargs -0 -- /bin/chmod -h u\+w
/usr/bin/xargs -0 -- /usr/bin/xattr -w com.apple.quarantine 0181\;61f2a908\;Homebrew\\x20Cask\;FE4A4113-733F-4447-B161-CFD9432C1249
==> Creating metadata directory: /opt/pkgmgr/homebrew/Caskroom/biglybt/.metadata/2.9.0.0/20220127142741.908
==> Creating metadata subdirectory: /opt/pkgmgr/homebrew/Caskroom/biglybt/.metadata/2.9.0.0/20220127142741.908/Casks
==> Installing artifacts
==> 3 artifacts defined
#<SortedSet: {#<Cask::Artifact::PreflightBlock:0x00007fd52f3095b0 @cask=#<Cask::Cask:0x00007fd53208bce0 @token="biglybt", @sourcefile_path=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/biglybt.rb>, @source="cask \"biglybt\" do\n  version \"2.9.0.0\"\n  sha256 \"efab84c5c1887dc1ece21b8e9b98be70556f74b4a6c62a0d3cb5dabf3695e54c\"\n\n  url \"https://github.com/BiglySoftware/BiglyBT/releases/download/v\#{version}/GitHub_BiglyBT_Installer.dmg\",\n      verified: \"github.com/BiglySoftware/BiglyBT/\"\n  name \"biglybt\"\n  desc \"Bittorrent client based on the Azureus open source project\"\n  homepage \"https://www.biglybt.com/\"\n\n  depends_on macos: \">= :el_capitan\"\n\n  preflight do\n    system_command \"\#{staged_path}/BiglyBT Installer.app/Contents/MacOS/JavaApplicationStub\",\n                   args:         [\n                     \"-dir\", \"\#{appdir}/BiglyBT\",\n                     \"-q\",\n                     \"-Dinstall4j.suppressStdout=true\",\n                     \"-Dinstall4j.debug=false\",\n                     \"-VcreateDesktopLinkAction$Boolean=false\",\n                     \"-VaddToDockAction$Boolean=false\"\n                   ],\n                   print_stderr: false\n  end\n\n  uninstall delete: \"\#{appdir}/BiglyBT\"\n\n  zap trash: [\n    \"~/Library/Application Support/BiglyBT\",\n    \"~/Library/Preferences/com.biglybt.plist\",\n    \"~/Library/Saved Application State/com.biglybt.savedState\",\n  ]\n\n  caveats do\n    depends_on_java \"8+\"\n  end\nend\n", @tap=#<Tap:0x00007fd52f38bb50 @user="Homebrew", @repo="cask", @name="homebrew/cask", @full_name="Homebrew/homebrew-cask", @path=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask>, @alias_table=nil, @alias_reverse_table=nil, @formula_renames={}, @cask_dir=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks>>, @block=#<Proc:0x00007fd53208bc40@/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/biglybt.rb:1>, @default_config=#<Cask::Config:0x00007fd52f3b0068 @explicit={}, @env={}, @default={:languages=>["en-IN"], :appdir=>#<Pathname:/Applications>, :colorpickerdir=>#<Pathname:/Users/admin/Library/ColorPickers>, :prefpanedir=>#<Pathname:/Users/admin/Library/PreferencePanes>, :qlplugindir=>#<Pathname:/Users/admin/Library/QuickLook>, :mdimporterdir=>#<Pathname:/Users/admin/Library/Spotlight>, :dictionarydir=>#<Pathname:/Users/admin/Library/Dictionaries>, :fontdir=>#<Pathname:/Users/admin/Library/Fonts>, :servicedir=>#<Pathname:/Users/admin/Library/Services>, :input_methoddir=>#<Pathname:/Users/admin/Library/Input Methods>, :internet_plugindir=>#<Pathname:/Users/admin/Library/Internet Plug-Ins>, :audio_unit_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/Components>, :vst_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST>, :vst3_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST3>, :screen_saverdir=>#<Pathname:/Users/admin/Library/Screen Savers>}>, @caskroom_path=#<Pathname:/opt/pkgmgr/homebrew/Caskroom/biglybt>, @metadata_main_container_path=#<Pathname:/opt/pkgmgr/homebrew/Caskroom/biglybt/.metadata>, @config=#<Cask::Config:0x00007fd52f30bb80 @explicit={}, @env={}, @default={:languages=>["en-IN"], :appdir=>#<Pathname:/Applications>, :colorpickerdir=>#<Pathname:/Users/admin/Library/ColorPickers>, :prefpanedir=>#<Pathname:/Users/admin/Library/PreferencePanes>, :qlplugindir=>#<Pathname:/Users/admin/Library/QuickLook>, :mdimporterdir=>#<Pathname:/Users/admin/Library/Spotlight>, :dictionarydir=>#<Pathname:/Users/admin/Library/Dictionaries>, :fontdir=>#<Pathname:/Users/admin/Library/Fonts>, :servicedir=>#<Pathname:/Users/admin/Library/Services>, :input_methoddir=>#<Pathname:/Users/admin/Library/Input Methods>, :internet_plugindir=>#<Pathname:/Users/admin/Library/Internet Plug-Ins>, :audio_unit_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/Components>, :vst_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST>, :vst3_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST3>, :screen_saverdir=>#<Pathname:/Users/admin/Library/Screen Savers>}>, @dsl=#<Cask::DSL:0x00007fd52f30b8d8 @cask=#<Cask::Cask:0x00007fd53208bce0 ...>, @token="biglybt", @version="2.9.0.0", @sha256=#<Checksum:0x00007fd52f30b180 @hexdigest="efab84c5c1887dc1ece21b8e9b98be70556f74b4a6c62a0d3cb5dabf3695e54c">, @url=#<URL::DSL:0x00007fd52f30a528 @uri=#<URI::HTTPS https://github.com/BiglySoftware/BiglyBT/releases/download/v2.9.0.0/GitHub_BiglyBT_Installer.dmg>, @verified="github.com/BiglySoftware/BiglyBT/", @using=nil, @tag=nil, @branch=nil, @revisions=nil, @revision=nil, @trust_cert=nil, @cookies=nil, @referer=nil, @header=nil, @user_agent=:default, @data=nil, @specs={:verified=>"github.com/BiglySoftware/BiglyBT/", :user_agent=>:default}>, @name=["biglybt"], @desc="Bittorrent client based on the Azureus open source project", @homepage="https://www.biglybt.com/", @depends_on={:macos=>#<MacOSRequirement: version>="10.11" []>}, @artifacts=#<SortedSet: {...}>, @caveats=#<Cask::DSL::Caveats:0x00007fd52f311aa8 @cask=#<Cask::Cask:0x00007fd53208bce0 ...>, @command=SystemCommand, @built_in_caveats={[:depends_on_java, "8+"]=>"biglybt requires Java 8+. You can install the latest version with:\n  brew install --cask temurin\n"}, @custom_caveats=[], @discontinued=false>, @language_eval=nil>>, @directives={:preflight=>#<Proc:0x00007fd52f309678@/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/biglybt.rb:13>}>, #<Cask::Artifact::Uninstall:0x00007fd52f312a70 @cask=#<Cask::Cask:0x00007fd53208bce0 @token="biglybt", @sourcefile_path=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/biglybt.rb>, @source="cask \"biglybt\" do\n  version \"2.9.0.0\"\n  sha256 \"efab84c5c1887dc1ece21b8e9b98be70556f74b4a6c62a0d3cb5dabf3695e54c\"\n\n  url \"https://github.com/BiglySoftware/BiglyBT/releases/download/v\#{version}/GitHub_BiglyBT_Installer.dmg\",\n      verified: \"github.com/BiglySoftware/BiglyBT/\"\n  name \"biglybt\"\n  desc \"Bittorrent client based on the Azureus open source project\"\n  homepage \"https://www.biglybt.com/\"\n\n  depends_on macos: \">= :el_capitan\"\n\n  preflight do\n    system_command \"\#{staged_path}/BiglyBT Installer.app/Contents/MacOS/JavaApplicationStub\",\n                   args:         [\n                     \"-dir\", \"\#{appdir}/BiglyBT\",\n                     \"-q\",\n                     \"-Dinstall4j.suppressStdout=true\",\n                     \"-Dinstall4j.debug=false\",\n                     \"-VcreateDesktopLinkAction$Boolean=false\",\n                     \"-VaddToDockAction$Boolean=false\"\n                   ],\n                   print_stderr: false\n  end\n\n  uninstall delete: \"\#{appdir}/BiglyBT\"\n\n  zap trash: [\n    \"~/Library/Application Support/BiglyBT\",\n    \"~/Library/Preferences/com.biglybt.plist\",\n    \"~/Library/Saved Application State/com.biglybt.savedState\",\n  ]\n\n  caveats do\n    depends_on_java \"8+\"\n  end\nend\n", @tap=#<Tap:0x00007fd52f38bb50 @user="Homebrew", @repo="cask", @name="homebrew/cask", @full_name="Homebrew/homebrew-cask", @path=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask>, @alias_table=nil, @alias_reverse_table=nil, @formula_renames={}, @cask_dir=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks>>, @block=#<Proc:0x00007fd53208bc40@/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/biglybt.rb:1>, @default_config=#<Cask::Config:0x00007fd52f3b0068 @explicit={}, @env={}, @default={:languages=>["en-IN"], :appdir=>#<Pathname:/Applications>, :colorpickerdir=>#<Pathname:/Users/admin/Library/ColorPickers>, :prefpanedir=>#<Pathname:/Users/admin/Library/PreferencePanes>, :qlplugindir=>#<Pathname:/Users/admin/Library/QuickLook>, :mdimporterdir=>#<Pathname:/Users/admin/Library/Spotlight>, :dictionarydir=>#<Pathname:/Users/admin/Library/Dictionaries>, :fontdir=>#<Pathname:/Users/admin/Library/Fonts>, :servicedir=>#<Pathname:/Users/admin/Library/Services>, :input_methoddir=>#<Pathname:/Users/admin/Library/Input Methods>, :internet_plugindir=>#<Pathname:/Users/admin/Library/Internet Plug-Ins>, :audio_unit_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/Components>, :vst_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST>, :vst3_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST3>, :screen_saverdir=>#<Pathname:/Users/admin/Library/Screen Savers>}>, @caskroom_path=#<Pathname:/opt/pkgmgr/homebrew/Caskroom/biglybt>, @metadata_main_container_path=#<Pathname:/opt/pkgmgr/homebrew/Caskroom/biglybt/.metadata>, @config=#<Cask::Config:0x00007fd52f30bb80 @explicit={}, @env={}, @default={:languages=>["en-IN"], :appdir=>#<Pathname:/Applications>, :colorpickerdir=>#<Pathname:/Users/admin/Library/ColorPickers>, :prefpanedir=>#<Pathname:/Users/admin/Library/PreferencePanes>, :qlplugindir=>#<Pathname:/Users/admin/Library/QuickLook>, :mdimporterdir=>#<Pathname:/Users/admin/Library/Spotlight>, :dictionarydir=>#<Pathname:/Users/admin/Library/Dictionaries>, :fontdir=>#<Pathname:/Users/admin/Library/Fonts>, :servicedir=>#<Pathname:/Users/admin/Library/Services>, :input_methoddir=>#<Pathname:/Users/admin/Library/Input Methods>, :internet_plugindir=>#<Pathname:/Users/admin/Library/Internet Plug-Ins>, :audio_unit_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/Components>, :vst_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST>, :vst3_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST3>, :screen_saverdir=>#<Pathname:/Users/admin/Library/Screen Savers>}>, @dsl=#<Cask::DSL:0x00007fd52f30b8d8 @cask=#<Cask::Cask:0x00007fd53208bce0 ...>, @token="biglybt", @version="2.9.0.0", @sha256=#<Checksum:0x00007fd52f30b180 @hexdigest="efab84c5c1887dc1ece21b8e9b98be70556f74b4a6c62a0d3cb5dabf3695e54c">, @url=#<URL::DSL:0x00007fd52f30a528 @uri=#<URI::HTTPS https://github.com/BiglySoftware/BiglyBT/releases/download/v2.9.0.0/GitHub_BiglyBT_Installer.dmg>, @verified="github.com/BiglySoftware/BiglyBT/", @using=nil, @tag=nil, @branch=nil, @revisions=nil, @revision=nil, @trust_cert=nil, @cookies=nil, @referer=nil, @header=nil, @user_agent=:default, @data=nil, @specs={:verified=>"github.com/BiglySoftware/BiglyBT/", :user_agent=>:default}>, @name=["biglybt"], @desc="Bittorrent client based on the Azureus open source project", @homepage="https://www.biglybt.com/", @depends_on={:macos=>#<MacOSRequirement: version>="10.11" []>}, @artifacts=#<SortedSet: {...}>, @caveats=#<Cask::DSL::Caveats:0x00007fd52f311aa8 @cask=#<Cask::Cask:0x00007fd53208bce0 ...>, @command=SystemCommand, @built_in_caveats={[:depends_on_java, "8+"]=>"biglybt requires Java 8+. You can install the latest version with:\n  brew install --cask temurin\n"}, @custom_caveats=[], @discontinued=false>, @language_eval=nil>>, @directives={:delete=>"/Applications/BiglyBT", :signal=>[]}>, #<Cask::Artifact::Zap:0x00007fd52f311ee0 @cask=#<Cask::Cask:0x00007fd53208bce0 @token="biglybt", @sourcefile_path=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/biglybt.rb>, @source="cask \"biglybt\" do\n  version \"2.9.0.0\"\n  sha256 \"efab84c5c1887dc1ece21b8e9b98be70556f74b4a6c62a0d3cb5dabf3695e54c\"\n\n  url \"https://github.com/BiglySoftware/BiglyBT/releases/download/v\#{version}/GitHub_BiglyBT_Installer.dmg\",\n      verified: \"github.com/BiglySoftware/BiglyBT/\"\n  name \"biglybt\"\n  desc \"Bittorrent client based on the Azureus open source project\"\n  homepage \"https://www.biglybt.com/\"\n\n  depends_on macos: \">= :el_capitan\"\n\n  preflight do\n    system_command \"\#{staged_path}/BiglyBT Installer.app/Contents/MacOS/JavaApplicationStub\",\n                   args:         [\n                     \"-dir\", \"\#{appdir}/BiglyBT\",\n                     \"-q\",\n                     \"-Dinstall4j.suppressStdout=true\",\n                     \"-Dinstall4j.debug=false\",\n                     \"-VcreateDesktopLinkAction$Boolean=false\",\n                     \"-VaddToDockAction$Boolean=false\"\n                   ],\n                   print_stderr: false\n  end\n\n  uninstall delete: \"\#{appdir}/BiglyBT\"\n\n  zap trash: [\n    \"~/Library/Application Support/BiglyBT\",\n    \"~/Library/Preferences/com.biglybt.plist\",\n    \"~/Library/Saved Application State/com.biglybt.savedState\",\n  ]\n\n  caveats do\n    depends_on_java \"8+\"\n  end\nend\n", @tap=#<Tap:0x00007fd52f38bb50 @user="Homebrew", @repo="cask", @name="homebrew/cask", @full_name="Homebrew/homebrew-cask", @path=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask>, @alias_table=nil, @alias_reverse_table=nil, @formula_renames={}, @cask_dir=#<Pathname:/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks>>, @block=#<Proc:0x00007fd53208bc40@/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/biglybt.rb:1>, @default_config=#<Cask::Config:0x00007fd52f3b0068 @explicit={}, @env={}, @default={:languages=>["en-IN"], :appdir=>#<Pathname:/Applications>, :colorpickerdir=>#<Pathname:/Users/admin/Library/ColorPickers>, :prefpanedir=>#<Pathname:/Users/admin/Library/PreferencePanes>, :qlplugindir=>#<Pathname:/Users/admin/Library/QuickLook>, :mdimporterdir=>#<Pathname:/Users/admin/Library/Spotlight>, :dictionarydir=>#<Pathname:/Users/admin/Library/Dictionaries>, :fontdir=>#<Pathname:/Users/admin/Library/Fonts>, :servicedir=>#<Pathname:/Users/admin/Library/Services>, :input_methoddir=>#<Pathname:/Users/admin/Library/Input Methods>, :internet_plugindir=>#<Pathname:/Users/admin/Library/Internet Plug-Ins>, :audio_unit_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/Components>, :vst_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST>, :vst3_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST3>, :screen_saverdir=>#<Pathname:/Users/admin/Library/Screen Savers>}>, @caskroom_path=#<Pathname:/opt/pkgmgr/homebrew/Caskroom/biglybt>, @metadata_main_container_path=#<Pathname:/opt/pkgmgr/homebrew/Caskroom/biglybt/.metadata>, @config=#<Cask::Config:0x00007fd52f30bb80 @explicit={}, @env={}, @default={:languages=>["en-IN"], :appdir=>#<Pathname:/Applications>, :colorpickerdir=>#<Pathname:/Users/admin/Library/ColorPickers>, :prefpanedir=>#<Pathname:/Users/admin/Library/PreferencePanes>, :qlplugindir=>#<Pathname:/Users/admin/Library/QuickLook>, :mdimporterdir=>#<Pathname:/Users/admin/Library/Spotlight>, :dictionarydir=>#<Pathname:/Users/admin/Library/Dictionaries>, :fontdir=>#<Pathname:/Users/admin/Library/Fonts>, :servicedir=>#<Pathname:/Users/admin/Library/Services>, :input_methoddir=>#<Pathname:/Users/admin/Library/Input Methods>, :internet_plugindir=>#<Pathname:/Users/admin/Library/Internet Plug-Ins>, :audio_unit_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/Components>, :vst_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST>, :vst3_plugindir=>#<Pathname:/Users/admin/Library/Audio/Plug-Ins/VST3>, :screen_saverdir=>#<Pathname:/Users/admin/Library/Screen Savers>}>, @dsl=#<Cask::DSL:0x00007fd52f30b8d8 @cask=#<Cask::Cask:0x00007fd53208bce0 ...>, @token="biglybt", @version="2.9.0.0", @sha256=#<Checksum:0x00007fd52f30b180 @hexdigest="efab84c5c1887dc1ece21b8e9b98be70556f74b4a6c62a0d3cb5dabf3695e54c">, @url=#<URL::DSL:0x00007fd52f30a528 @uri=#<URI::HTTPS https://github.com/BiglySoftware/BiglyBT/releases/download/v2.9.0.0/GitHub_BiglyBT_Installer.dmg>, @verified="github.com/BiglySoftware/BiglyBT/", @using=nil, @tag=nil, @branch=nil, @revisions=nil, @revision=nil, @trust_cert=nil, @cookies=nil, @referer=nil, @header=nil, @user_agent=:default, @data=nil, @specs={:verified=>"github.com/BiglySoftware/BiglyBT/", :user_agent=>:default}>, @name=["biglybt"], @desc="Bittorrent client based on the Azureus open source project", @homepage="https://www.biglybt.com/", @depends_on={:macos=>#<MacOSRequirement: version>="10.11" []>}, @artifacts=#<SortedSet: {...}>, @caveats=#<Cask::DSL::Caveats:0x00007fd52f311aa8 @cask=#<Cask::Cask:0x00007fd53208bce0 ...>, @command=SystemCommand, @built_in_caveats={[:depends_on_java, "8+"]=>"biglybt requires Java 8+. You can install the latest version with:\n  brew install --cask temurin\n"}, @custom_caveats=[], @discontinued=false>, @language_eval=nil>>, @directives={:trash=>["~/Library/Application Support/BiglyBT", "~/Library/Preferences/com.biglybt.plist", "~/Library/Saved Application State/com.biglybt.savedState"], :signal=>[]}>}>
==> Installing artifact of class Cask::Artifact::PreflightBlock
/opt/pkgmgr/homebrew/Caskroom/biglybt/2.9.0.0/BiglyBT\ Installer.app/Contents/MacOS/JavaApplicationStub -dir /Applications/BiglyBT -q -Dinstall4j.suppressStdout=true -Dinstall4j.debug=false -VcreateDesktopLinkAction\$Boolean=false -VaddToDockAction\$Boolean=false
==> Purging files for version 2.9.0.0 of Cask biglybt
Error: Failure while executing; `/opt/pkgmgr/homebrew/Caskroom/biglybt/2.9.0.0/BiglyBT\ Installer.app/Contents/MacOS/JavaApplicationStub -dir /Applications/BiglyBT -q -Dinstall4j.suppressStdout=true -Dinstall4j.debug=false -VcreateDesktopLinkAction\$Boolean=false -VaddToDockAction\$Boolean=false` exited with 1. Here's the output:
2022-01-27 19:57:43.062 JavaApplicationStub[5391:125383] +[NSXPCSharedListener endpointForReply:withListenerName:]: an error occurred while attempting to obtain endpoint for listener 'ClientCallsAuxiliary': Connection interrupted
2022-01-27 19:57:47.848 JavaApplicationStub[5391:125354] In -[NSApplication(NSQuietSafeQuit) _updateCanQuitQuietlyAndSafely], _LSSetApplicationInformationItem(NSCanQuitQuietlyAndSafely) returned error -600

/opt/pkgmgr/homebrew/Library/Homebrew/system_command.rb:302:in `assert_success!'
/opt/pkgmgr/homebrew/Library/Homebrew/system_command.rb:65:in `run!'
/opt/pkgmgr/homebrew/Library/Homebrew/system_command.rb:40:in `run'
/opt/pkgmgr/homebrew/Library/Homebrew/system_command.rb:44:in `run!'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/dsl/base.rb:22:in `system_command'
/opt/pkgmgr/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/biglybt.rb:14:in `block (2 levels) in load'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/artifact/abstract_flight_block.rb:45:in `instance_eval'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/artifact/abstract_flight_block.rb:45:in `abstract_phase'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/artifact/abstract_flight_block.rb:28:in `install_phase'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/installer.rb:240:in `block in install_artifacts'
/opt/pkgmgr/homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/lib/ruby/2.6.0/set.rb:777:in `each'
/opt/pkgmgr/homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/lib/ruby/2.6.0/set.rb:777:in `each'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/installer.rb:233:in `install_artifacts'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/installer.rb:116:in `install'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/cmd/install.rb:71:in `block in install_casks'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/cmd/install.rb:70:in `each'
/opt/pkgmgr/homebrew/Library/Homebrew/cask/cmd/install.rb:70:in `install_casks'
/opt/pkgmgr/homebrew/Library/Homebrew/cmd/install.rb:179:in `install'
/opt/pkgmgr/homebrew/Library/Homebrew/brew.rb:110:in `<main>'

**Comment by neutric** [comment]
This lacks context necessary to analyze what might be going wrong. Please file an [issue](https://github.com/Homebrew/homebrew-cask/issues/new?assignees=&labels=&template=01_bug_report.yml), which will guide you regarding necessary information to provide.
